### PR TITLE
Improve orgs integration

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by fvanderbiest** [comment]
I guess we already have a translation for `password.label.empty`. Would it be more appropriate with a dedicated `org.label.empty` ?

**Comment by Vampouille** [comment]
This commit ad83eda fix orgs missing in ogc services stats.

File: ldapadmin/src/main/java/org/georchestra/ldapadmin/ds/OrgsDao.java [hunk 1]
```diff
@@ -90,6 +91,17 @@ public Org findByCommonName(String commonName) {
     }
 
     /**
+     * Search for organization extension with specified identifier
+     * @param cn distinguish name of organization for example : 'psc' to retreive
```
Comment (pmauduit) [inline ldapadmin/src/main/java/org/georchestra/ldapadmin/ds/OrgsDao.java]: *retrieve

File: ldapadmin/src/main/webapp/WEB-INF/i18n/application_de.properties [hunk 2]
```diff
@@ -77,6 +77,14 @@ firstName.placeholder=Vorname
 firstName.error.required=Notwendig
 org.label=Organisation
 org.placeholder=Organisation
+org.empty=leer
+org.cannot_find_org_in_list=My organization is not present
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/WEB-INF/i18n/application_de.properties]: you should ask Tobias ... ;-)

File: ldapadmin/src/main/webapp/WEB-INF/i18n/application_fr.properties [hunk 3]
```diff
@@ -77,6 +77,14 @@ firstName.placeholder=Prénom
 firstName.error.required=Requis
 org.label=Organisme
 org.placeholder=Organisme
+org.empty=vide
+org.cannot_find_org_in_list=Mon organisme n'apparait pas dans la liste
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/WEB-INF/i18n/application_fr.properties]: apparaît

File: ldapadmin/src/main/webapp/WEB-INF/views/createAccountForm.jsp [hunk 4]
```diff
@@ -195,7 +232,56 @@
...
+
+		// submit org creation and fill droplist with new created org
+		$("#create_org_button").click(function (event) {
+			event.preventDefault();
+			var valid = testField("orgName");
+			valid = testField("orgShortName") && valid;
+			valid = testField("orgAddress") && valid;
+			valid = testField("orgType") && valid;
+
+			if(!valid)
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/WEB-INF/views/createAccountForm.jsp]: parenthesis required (coding style): `if () {}`

File: ldapadmin/src/main/webapp/WEB-INF/views/validation.jsp [hunk 5]
```diff
@@ -228,6 +228,8 @@ function isUidValid(uid) {
  * @returns {boolean}
  */
 function isNotEmpty(str) {
+    if(str == "-")
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/WEB-INF/views/validation.jsp]: parenthesis required

**Comment by fvanderbiest** [comment]
Minor comments, LGTM otherwise.

**Comment by fvanderbiest** [comment]
Please merge once you have fixed them. 
Thanks !

**Pull request merged**